### PR TITLE
Bump version to 0.2.6

### PR DESCRIPTION
**Pull request by masayag**

### CODE REVIEW

File: charts/workflows/charts/escalation/templates/01-configmap_01-ticketescalation-resources.yaml
Comment (gabriel-farache) [inline charts/workflows/charts/escalation/templates/01-configmap_01-ticketescalation-resources.yaml:1]: why are those files deleted?

Comment (masayag) [reply charts/workflows/charts/escalation/templates/01-configmap_01-ticketescalation-resources.yaml:1]: The escalation is being deployed by kustomize:
https://github.com/parodos-dev/serverless-workflows-helm/tree/main/kustomize/escalation/base/config

We shouldn't support it via helm nomore.

Comment (gabriel-farache) [reply charts/workflows/charts/escalation/templates/01-configmap_01-ticketescalation-resources.yaml:1]: ok so you start cleaning what is already merged but before all workflows have the new way of doing automatically pushed 
You like mixing stuff ^^

Comment (masayag) [reply charts/workflows/charts/escalation/templates/01-configmap_01-ticketescalation-resources.yaml:1]: pretty much. I failed to build the chart since it complains of not having the Chart.yaml under the escalation folder.
I believe I merged an automated PR which re-created the escalation folder and now I'm cleaning it to make the `helm dep update workflows` finish successfully.

Comment (gabriel-farache) [reply charts/workflows/charts/escalation/templates/01-configmap_01-ticketescalation-resources.yaml:1]: ok

**Review by gabriel-farache — APPROVED** [review]
